packages/credential-router/__tests__/action-handler.test.ts

```diff
@@ -159,6 +208,29 @@ describe('CredentialRouter', () => {
         'invalid_setup: your agent does not seem to have CredentialW3c plugin installed',
       );
     });
+
+    it(`should throw error when create a credential router with proofFormat is EnvelopingProofJose`, async () => {
+      const mockIssuerAgentContext = {
+        agent: {
+          createVerifiableCredential: undefined,
+        },
+      } as unknown as IssuerAgentContext;
+
+      const credentialPlugin = new CredentialRouter();
+      const args: ICreateVerifiableCredentialArgs = {
+        credential: RAW_CREDENTIAL,
+        proofFormat: 'EnvelopingProofJose',
+      };
+
+      await expect(
+        credentialPlugin.routeCreationVerifiableCredential(
+          args,
+          mockIssuerAgentContext,
+        ),
+      ).rejects.toThrow(
+        'invalid_setup: your agent does not seem to have CredentialW3c plugin installed',
+      );
+    });
```

Comment:
I realized you're no longer using the `EnvelopingProofJose` proof format in the `credential-router` plugin, so I think you can remove any test cases related to the `EnvelopingProofJose` proof format here.